openshift/4.0/pipelines/Jenkinsfile.cicd

```diff
@@ -219,14 +219,15 @@ pipeline {
         script {
           def img_backend = "${APP_NAME}-api"
           def img_frontend = "${APP_NAME}-app"
+          def img_tag = "latest-${DESTINATION}" // image tag on build config is based on environment, e.g. latest-dev, latest-test, latest-uat
 
           // tag release version (if found)
           if (RELEASE_VERSION != '') {
-            echo "Tagging images - ${img_backend}:latest, ${img_frontend}:latest..."
-            sh "oc tag ${img_backend}:latest ${img_backend}:${RELEASE_VERSION}"
-            sh "oc tag ${img_frontend}:latest ${img_frontend}:${RELEASE_VERSION}"
+            echo "Tagging images - ${img_backend}:latest-env, ${img_frontend}:latest-env"
+            sh "oc tag ${img_backend}:${img_tag} ${img_backend}:${RELEASE_VERSION}"
```

Comment:
This retags :latest-${DESTINATION} to :${RELEASE_VERSION}
but I'm not seeing what is creating the :latest-${DESTINATION} 
tag in the first place.

Reply:
The tag **latest-${DESTINATION}** is created by the build config.
The tag is defined in the build.yaml template with the parameter OUTPUT_IMAGE_TAG. Are the parameter values stored somewhere? If not then there's no code to update. 

For reference, the api template is here: openshift\4.0\templates\api\build.yaml

Reply:
I think the template provides default values. My guess is that Devin is requesting the default be changed in the **build.yaml** template

Reply:
Also please make sure to modify the existing build configs right after this gets merged or else the pipeline will fail